firstvoices/backend/tests/test_apis/test_story_api.py

```diff
@@ -70,17 +71,30 @@ def assert_updated_instance(self, expected_data, actual_instance: Story):
 
     def assert_update_response(self, expected_data, actual_response):
         assert actual_response["title"] == expected_data["title"]
-        assert (
-            actual_response["relatedAudio"][0]["id"] == expected_data["relatedAudio"][0]
+
+        response_related_audio_ids = []
+        for i in actual_response["relatedAudio"]:
+            response_related_audio_ids.append(i["id"])
```

Comment:
A list comprehension might be a simpler way to write this:
`response_related_audio_ids = [ i["id] for i in actual_response["relatedAudio"] ]`